tools/admin/src/main/scala/whisk/core/database/UserCommand.scala

```diff
@@ -164,14 +172,23 @@ class UserCommand extends Subcommand("user") with WhiskCommand {
     authStore
       .get[ExtendedAuth](DocInfo(create.subject()))
       .flatMap { auth =>
+        val nsToUpdate = create.desiredNamespace(authKey).name
+        var newNS = auth.namespaces.filter(_.namespace.name != nsToUpdate)
         if (auth.isBlocked) {
           Future.successful(Left(IllegalState(CommandMessages.subjectBlocked)))
-        } else if (auth.namespaces.exists(_.namespace.name == create.desiredNamespace(authKey).name)) {
-          Future.successful(Left(IllegalState(CommandMessages.namespaceExists)))
-        } else {
-          val newNS = auth.namespaces + WhiskNamespace(create.desiredNamespace(authKey), authKey)
+        } else if (!auth.namespaces.exists(_.namespace.name == nsToUpdate) || create.force.isSupplied) {
+          newNS = newNS.+(WhiskNamespace(create.desiredNamespace(authKey), authKey))
```

Comment:
Instead of `.+(` just simply use `+` like `newNS + WhiskNamespace(create.desiredNamespace(newAuthKey), newAuthKey)`